llvm/test/CodeGen/AMDGPU/GlobalISel/extractelement.ll

```diff
@@ -2714,6 +2714,168 @@ entry:
   ret double %ext
 }
 
+define amdgpu_ps double @dyn_extract_v7f64_s_v_bitcast(<14 x float> inreg %userData, i32 %sel) {
```

Comment:
Can you precommit these tests, or do they fail to compile without your patch?

Reply:
They fail without this patch